src/current/v24.1/changefeed-sinks.md

```diff
@@ -495,13 +496,14 @@ The following are considerations when using the webhook sink:
 
 * Only supports HTTPS. Use the [`insecure_tls_skip_verify`]({% link {{ page.version.version }}/create-changefeed.md %}#insecure-tls-skip-verify) parameter when testing to disable certificate verification; however, this still requires HTTPS and certificates.
 * Supports JSON output format. You can use the [`format=csv`]({% link {{ page.version.version }}/create-changefeed.md %}#format) option in combination with [`initial_scan='only'`]({% link {{ page.version.version }}/create-changefeed.md %}#initial-scan) for CSV-formatted messages.
-* There is no concurrency configurability.
+* Supports concurrency configuration. Set the [`changefeed.sink_io_workers` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-changefeed-sink-io-workers) to configure the number of concurrent workers used by changefeeds in the cluster when sending requests to a webhook sink.
```

Comment:
the structure of the change appears different in this file IIUC

in the other files you removed "there is no concurrency config", whereas in this file you removed that and added this line

technical reasons? or writing reasons?

Reply:
This was a remnant of me drafting, thanks for catching Rich!